### ISSUE TITLE
Minor annoyance when testing the ClusterGrainHelloWorld example 

### ISSUE THREAD

**Issue by chezuma** [AsynkronIT/protoactor-dotnet#742]
## Which build script
./examples/ClusterGrainHelloWorld/run.sh

## Expected Behavior
Expects the containers to be built and run by docker-compose regardless of current dotnet runtime. 

## Actual Behavior
Building node1
Step 1/6 : FROM mcr.microsoft.com/dotnet/core/aspnet:3.1.0-alpine3.10
 ---> 21eae2e8220f
Step 2/6 : RUN echo 'http://dl-cdn.alpinelinux.org/alpine/v3.8/main' >> /etc/apk/repositories &&     apk update --no-cache &&     apk add --no-cache bash libc6-compat=1.1.19-r11
 ---> Using cache
 ---> 12531e74fb6b
Step 3/6 : ADD ./bin/Release/netcoreapp3.1/publish/ /app
ERROR: Service 'node1' failed to build : ADD failed: file not found in build context or excluded by .dockerignore: stat bin/Release/netcoreapp3.1/publish/: file does not exist

## Steps to Reproduce the Problem

  1. Do not have dotnet 3.1 as default 
  2. run the run.sh in ClusterGrainHelloWorld

## Solution 
Create the build step inside the Dockerfile instead of running building the projects in the script file

**Comment by alexeyzimarev** [comment]
Hey man, nice to see you here :) Definitely, some examples still wait for the .NET 5 upgrade. We can also remove the need for libcompat.

**Comment by chezuma** [comment]
Hey, thanks for a swift reply :) 

The ClusterGrainHelloWorld example is already NET 5.0, only the run.sh which referenced the Core 3.1 path. 

I fixed up the dockerfiles, they now include the build step. Updated the docker-compose.yml file to set the correct context so the dockerfiles can reach files in ./src of the project. 

Made a branch, tried to push, but it seems that I do not have permission to do this :/
